xml/admin_saltcluster.xml

```diff
@@ -1181,28 +1196,33 @@ OSD_ID  HOST         STATE                    PG_COUNT  REPLACE  FORCE  STARTED_
    </step>
    <step>
     <para>
-     On the &smaster;, install the <package>ceph-salt</package> package if it
-     is not already installed (see <xref linkend="deploy-cephadm-cephsalt"/>),
-     and include the new &adm; in the cluster as described in
-     <xref linkend="deploy-cephadm-configure-minions"/> and
-     <xref linkend="deploy-cephadm-configure-admin"/>.
-    </para>
-   </step>
-   <step>
-    <para>
-     Import the backed up cluster configuration and apply it:
+     Install &cephsalt; on the new admin (&smaster;) node:
     </para>
 <screen>
-&prompt.smaster;ceph-salt import <replaceable>CLUSTER_CONFIG</replaceable>.json
-&prompt.smaster;ceph-salt apply
+&prompt.smaster;zypper install ceph-salt
+&prompt.smaster;systemctl restart salt-master.service
+&prompt.smaster;salt \* saltutil.sync_all
```

Comment:
both syntaxes work fine, but I think it's more common to see `salt '*' ..` syntax instead of `salt \* ` syntax (at least in this documentation?).